op-enable.go

```diff
@@ -34,6 +35,13 @@ const (
 	dockerCaCert  = "ca.pem"
 	dockerSrvCert = "cert.pem"
 	dockerSrvKey  = "key.pem"
+
+	dockerUrlGlobal       = "https://get.docker.com/"
```

Comment:
and introduce an `azureEnvironment` optional public setting that defaults to `AzureCloud` and can be configured as `AzureChina`. so using this string, you would look up at a `map[string]struct{dockerUrl, composeUrl string}`. This way you would using `if endpoint, err := util.GetAzureEndpoint(); err == nil && endpoint == azureEndpointChina {` over and over again.